docs/docs/addons-test-utils.md

```diff
@@ -53,20 +53,24 @@ Call [`createRenderer()`](#createrenderer) in your tests to create a shallow ren
 
 After `shallowRenderer.render()` has been called, you can use [`shallowRenderer.getRenderOutput()`](#shallowrenderer.getrenderoutput) to get the shallowly rendered output.
 
-You can then begin to assert facts about the output. For example, if your component's render method returns:
+You can then begin to assert facts about the output. For example, if you have the following component:
 
 ```javascript
-<div>
-  <span className="heading">Title</span>
-  <Subcomponent foo="bar" />
-</div>
+const MyComponent = () => (
```

Comment:
Could you please use regular function syntax here, as we do elsewhere in the docs?